README.md

```diff
@@ -23,7 +23,6 @@ available language server do.
   - If using [vim-plug][3]: `Plug 'mfussenegger/nvim-lint'`
   - If using [packer.nvim][4]: `use 'mfussenegger/nvim-lint'`
 
-
```

Comment:
Please no unrelated style changes

Reply:
i am sorry. i've just revertes the changes!